code/modules/mob/living/simple_animal/slaughter/slaughter.dm

```diff
@@ -0,0 +1,274 @@
+///The following mini-antag was coded by 	KorPhaeron   for TG Station
+//Credits: Concept: KorPhaeron, Sprite:Ausops, Sounds:Cuboos
+///Ported for paradise by Aurorablade/Fethas
+///Things added for port:
+//Various tweeks
+//Blood...
+/// Made the message tell you waht the demon is sinking into(gibs/blood etc)
+///Alot of blood...Alot..Of blood..
+
+//////////////////The Monster
+
+/mob/living/simple_animal/slaughter
+	name = "Slaughter Demon"
+	real_name = "Slaughter Demon"
+	desc = "You should run."
+	speak_emote = list("gurgles")
+	emote_hear = list("wails","screeches")
+	response_help  = "thinks better of touching"
+	response_disarm = "flails at"
+	response_harm   = "punches"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "daemon"
+	speed = 0
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	attack_sound = 'sound/misc/demon_attack1.ogg'
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2  = 0
+	max_n2  = 0
+	minbodytemp = 0
+	faction = list("slaughter")
+	attacktext = "wildly tears into"
+	maxHealth = 250
+	health = 250
+	environment_smash = 1
+	universal_understand = 1
+	melee_damage_lower = 15//Paradise port:This was also 30, but i moved it down.
+	melee_damage_upper = 30
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	var/devoured = 0
+	var/phased = FALSE
+	var/holder = null
+	var/eating = FALSE
+	var/mob/living/kidnapped = null
+	//var/cooldown = 0
+	var/playstyle_string = "<B>You are the Slaughter Demon, a terible creature from another existence. You have a single desire: To kill.  \
+						You may Ctrl+Click on blood pools to travel through them, appearing and dissaapearing from the station at will. \
+						Pulling a dead or critical mob while you enter a pool will pull them in with you, allowing you to feast. </B>"
+   //Paradise todo:change to altclick,add objectives. need ideas as i want people to be creative, not just RAWR KILL
+
+
+/mob/living/simple_animal/slaughter/New()
+	..()
+	spawn(5)
+	AddSpell(new /obj/effect/proc_holder/spell/wizard/targeted/slaughter_whisper)
```

Comment:
ehhh...not sure if this is the best way to handle it or if it should be added to the mind.